apps/design-tokens-ui/src/app/app.component.scss

```
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../../libs/barista-components/style/font-styles.scss';
```

You can import like this as you specified the `stylePreprocessorOptions` in the `angular.json`

```suggestion
@import 'font-styles.scss';
```

---

Sass can't find it. Am I missing something?

---

the style preprocessor stated in the angular.json only states `libs/barista-components/core/src/style/` and not `libs/barista-components/style/` so an import from `font-styles.scss` is not possible. 
Either leave the import as is or add the path to the includePaths in the angular.json

```
           "stylePreprocessorOptions": {
              "includePaths": ["libs/barista-components/core/src/style/"]
            },
```